src/cljs/main/broadfcui/auth.cljs

```diff
@@ -375,7 +375,9 @@
                                    :listen (constantly nil)}
                                   :signOut on-sign-out})]
                       (user/set-google-auth2-instance! auth2)
-                      (on-sign-in))
+                      (on-sign-in)
+                      (when (some? extra-on-sign-in)
+                        (extra-on-sign-in)))
```

Comment:
allow callers of `force-signed-in` to pass an additional callback function to run after sign-in succeeds. In this PR's use case, it allows us to wait for sign-in finish before redirecting the browser location .